payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayLauncherViewModel.kt

```diff
@@ -261,12 +263,18 @@ internal class GooglePayLauncherViewModel(
                 paymentAnalyticsRequestFactory = analyticsRequestFactory
             )
 
+            val errorReporter = RealErrorReporter(
```

Comment:
🤔 It might be better to use `ErrorReporter.createFallbackInstance` instead of creating this manually.

Manual creation everywhere will make it hard to refactor the implementation long term.